doc/source/getting_started/installing_extensions.rst

```diff
@@ -0,0 +1,165 @@
+.. _installing_odl_extensions:
+
+Compiled extensions
+===================
+There are several compiled extensions to ODL.
+Some of them can be installed using ``conda`` or ``pip``, others require manual compilation.
+This section assumes that you have a working installation of python and ODL.
+
+
+ASTRA for X-ray tomography
+==========================
+To calculate fast forward and backward projections for image reconstruction in X-ray tomography, install the `ASTRA tomography toolbox <https://github.com/astra-toolbox/astra-toolbox>`_.
+ASTRA projectors are fully supported in ODL.
+
+You can try using the conda package, but we can give no guarantee that it works out of the box:
+
+.. code-block:: bash
+
+    $ conda install -c astra-toolbox astra-toolbox
+
+For further instructions, check `the ASTRA GitHub page <https://github.com/astra-toolbox/astra-toolbox>`_.
+
+
+STIR for emission tomography
+============================
+For applications in emission tomography, i.e. PET or SPECT, install `STIR`_ with Python bindings.
+Support for STIR is currently very limited.
+
+
+CUDA backend for linear arrays
+==============================
+The `odlcuda`_ backend for fast array calculations on CUDA requires the `CUDA toolkit`_ (on Linux: use your distro package manager) and a CUDA capable graphics card with compute capability of at least 3.0.
+Search `this table <https://en.wikipedia.org/wiki/CUDA#GPUs_supported>`_ for your model.
+
+Installation using ``conda``
+----------------------------
+.. note::
+    In conda, the ``odlcuda`` package is currently available only for Linux 64-bit and Python 3.5.
+    Furthermore, you may experience failures due to "invalid device function" -- this is a known issue, and we're trying to fix it.
+
+If you have installed an ODL release, simply run (in a directory of your choice)
+
+.. code-block:: bash
+
+    $ conda install -c odlgroup odlcuda
+
+If you have installed ODL from source, you need to prevent conda from installing its version of ODL.
+To do this, find out the dependencies of ``odlcuda`` by running
+
+.. code-block:: bash
+
+    $ conda install --dry-run odlcuda
+
+Install all its dependencies except ``odl`` and ``odlcuda``.
+Finally, install ``odlcuda`` without dependencies:
+
+.. code-block:: bash
+
+    $ conda install --no-deps odlcuda
```

Comment:
I'd rather have it removed for now, it advises to install a broken package. The instructions to build using conda are still valid, so it's not that we give up.